client/landing/gutenboarding/components/signup-form/index.tsx

```diff
@@ -197,7 +197,7 @@ const SignupForm = ( { onRequestClose }: Props ) => {
 		signupUrl = addAnchorQueryParts( signupUrl );
 	}
 	signupUrl = encodeURIComponent( signupUrl );
-	const loginUrl = `/log-in/new${ langFragment }?redirect_to=${ loginRedirectUrl }&signup_url=${ signupUrl }`;
+	const loginUrl = `/log-in/new${ langFragment }?redirect_to=${ loginRedirectUrl }&signup_flow=${ signupUrl }`;
```

Comment:
This change gives me pause--I'm unclear if there are other places that expect `signup_url` as a query param.

Reply:
I dug through the commit history to try and understand this better. The `signup_flow` param was added in https://github.com/Automattic/wp-calypso/pull/40795, and passed down as a prop to `LoginLinks`... where it was used to render the signup form link.

However this use was removed recently in https://github.com/Automattic/wp-calypso/pull/50008, where the signupUrl is now [hard-coded](https://github.com/Automattic/wp-calypso/pull/50008/files#diff-af36267901f79bfab39d29ba249bbb3fb65fe581616b37a37bc5ea1e03e9ab10R100).

So it looks safe to remove the `signup_url` query parameter from here.

Reply:
I've been looking to find how `signup_flow` is used... it seems to have a specific meaning, that is which flow name to append to `/start`. One example is the actionbar, which then uses `signup_flow=account` for the login page, and then goes to `/start/account` (`account` is the "signup flow") when clicking on "Create a new account" under the login form:

<img width="600" alt="Screen Shot 2021-02-22 at 07 45 49" src="https://user-images.githubusercontent.com/1699996/108716688-0d546e00-74e2-11eb-90cd-e7f25007d42b.png">

So I'm thinking it'd be better to keep this param as `signup_url`, and then separately look for that query parameter in the `getSignupUrl` function, to keep the use of `signup_flow` specific to `/start` flows.